src/boost_fractor_recurprop.jl

```diff
@@ -311,37 +321,37 @@ function cheerleader(amin, nmax, bdry::SetupBoundaries, coords::CoordinateSystem
             # Do NOT Propagate to the fields in the current gap to the right
             # Region 1 always contains left-moving fields.
             # Propagate to the fields in the next gap to the left
-            fields[i+1,:,:] = prop(fields[i+1,:,:], bdry.distance[i+1], diskR, bdry.eps[i+1], bdry.relative_tilt_x[i+1], bdry.relative_tilt_y[i+1], bdry.relative_surfaces[i+1,:,:], lambda, coords)
```

Comment:
Since the _prop_ function is actually in place now, adding ! would be sensible?
See e.g. here: https://stackoverflow.com/questions/39293082/mutating-function-in-julia-function-that-modifies-its-arguments